app/pods/settings/emails/template.hbs

```diff
@@ -1 +1,45 @@
-{{outlet}}
\ No newline at end of file
+{{ page-title (t "manage-emails")}}
+
+<div class="auk-u-m-8">
+  <div class="auk-u-mb-3">
+    <Auk::Panel>
+      <Auk::Panel::Header>
+        <Auk::Toolbar>
+          <Auk::Toolbar::Group @position="left">
+            <Auk::Toolbar::Item>
+              <h4 class="auk-panel__title">{{t "manage-emails"}}</h4>
+            </Auk::Toolbar::Item>
+          </Auk::Toolbar::Group>
+        </Auk::Toolbar>
+      </Auk::Panel::Header>
+      <Auk::Panel::Body>
+        <Auk::Label class="auk-u-mb-1">
+          {{t "email-to-translationrequest"}}
+        </Auk::Label>
+        <Auk::Input
```

Comment:
Kan je proberen de gewone indentatie aan te houden hier (2 spaties)? Graag ook geen padding binnen `{{ }}`. Wanneer we tot de standaard ember linting rules komen zal dit in de toekomst ook meer geautomatiseerd kunnen denk ik.